src/app/_services/api.service.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('API Service', () => {
   it('should load specific country data', () => {
     const country = 'BE';
     const baseUrl = `${environment.serverAPI}/${service.suffixCountryHistoricalUrl}?country=${country}`;
-    spyOn(service, 'loadCountryData').and.callThrough();
+    const spyLoadCountryData = jest.spyOn(service, 'loadCountryData');
```

Comment:
Does it make sense to declare a variable? It doesn't seem to be used/accessed later?